Source/UI/NewUI/CommonUI/View/Core/SecureDisplayView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  SecureDisplayView.swift
+//  Frames
+//
+//  Created by Alex Ioja-Yang on 
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+/**
+Secure display view preventing leaking of content
+ */
+public final class SecureDisplayView: UIView {
```

Comment:
Does this need to be public, I am hoping the components we develop are using SecureDisplayView internally our merchants will not be interested to use SecureDisplayView from external.

Reply:
Also we need to add some unit tests here for SecureDisplayView and what is expected. we can refer the internal library for SecureDisplayView for same.